verticapy/jupyter/extensions/sql_magic.py

```diff
@@ -57,19 +57,606 @@ def sql_magic(
+        DROP TABLE IF EXISTS test;
+        CREATE TABLE test AS SELECT 'Badr Ouali' AS name;
+        SELECT * FROM test;
+
+    **Execution**: 0.05s
+
+    .. ipython:: python
+        :suppress:
+        :okwarning:
+
+        %%sql
+        DROP TABLE IF EXISTS test;
+        CREATE TABLE test AS SELECT 'Badr Ouali' AS name;
+        SELECT * FROM test;
+
+    .. ipython:: python
+        :suppress:
+
+        test_table = _
+        html_file = open("figures/jupyter_extensions_sql_magic_sql_magic_3.html", "w")
+        html_file.write(test_table._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_sql_magic_sql_magic_3.html
+
+    To add comments to a query, use one of the following comment syntaxes:
+
+    .. code-block:: python
+
+        %%sql
+        -- Comment Test
+        /* My Vertica Version */
+        SELECT version(); -- Select my current version
+
+    **Execution**: 0.005s
+
+    .. ipython:: python
+        :suppress:
+
+        %%sql
+        -- Comment Test
+        /* My Vertica Version */
+        SELECT version(); -- Select my current version
+
+    .. ipython:: python
+        :suppress:
+
+        test_comment = _
+        html_file = open("figures/jupyter_extensions_sql_magic_sql_magic_4.html", "w")
+        html_file.write(test_comment._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_sql_magic_sql_magic_4.html
+
+    Get the vDataFrame of a query
+    =============================
+
+    Results of a SQL Magic query are stored in a vDataFrame, which is assigned
+    to a temporary variable called '_'. You can assign this temporary varaible
+    to a new variable to save your results.
+
+    .. code-block:: python
+
+        %%sql
+        SELECT
+            age,
+            fare,
+            pclass
+        FROM titanic
+        WHERE age IS NOT NULL AND fare IS NOT NULL;
+
+    **Execution**: 0.007s
+
+    .. ipython:: python
+        :suppress:
+
+        %%sql
+        SELECT
+            age,
+            fare,
+            pclass
+        FROM titanic
+        WHERE age IS NOT NULL AND fare IS NOT NULL;
+
+    .. ipython:: python
+        :suppress:
+
+        titanic_clean = _
+        html_file = open("figures/jupyter_extensions_sql_magic_sql_magic_5.html", "w")
+        html_file.write(titanic_clean._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: figures/jupyter_extensions_sql_magic_sql_magic_5.html
+
+    Assign the results to a new variable:
+
+    .. code-block:: python
+
+        titanic_clean = _
+        display(titanic_clean)
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_sql_magic_sql_magic_5.html
+
+    Temporary results are stored in a vDataFrame, allowing you to call
+    vDataFrame methods:
+
+    .. ipython:: python
+
+        @savefig jupyter_extensions_sql_magic_sql_magic_corr.png
```

Comment:
be careful, it is only valid when using Matplotlib, but you're using Plotly. You should change the code otherwise it will not work.